### PR TITLE
update kubeadm doc

### PR DESCRIPTION
**Pull request by dejanu**
### Description

Update the overview to make it clear that `kubeadm join` can be run to create a worker node as well as a new control plane.

### Issue

Closes:  #48399

### CODE REVIEW

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ content_type: concept
 weight: 30
 ---
 <!-- overview -->
-This command initializes a Kubernetes worker node and joins it to the cluster.
+This command initializes a Kubernetes node and joins it to the cluster.
```
Comment (T-Lakshmi) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md:10]: WDYT?

```suggestion
This command initializes a new Kubernetes node and joins it to the existing cluster.
```

**Comment by T-Lakshmi** [comment]
[Preview page](https://deploy-preview-48400--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/kubeadm-join/)

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md (hunk 1 above)
Comment (neolit123) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md:10]: yes please use this exactly
https://github.com/kubernetes/website/issues/48399#issuecomment-2418771122

Comment (dejanu) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md]: Done.

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Review by mengjiao-liu — COMMENTED** [review]
/lgtm

**Review by dipesh-rawat — COMMENTED** [review]
/approve

**Pull request merged**